packages/manager/apps/dedicated/client/app/ip/components/list/byoip/delete/ip-ip-byoip-delete.html

```diff
@@ -5,8 +5,8 @@
         on-dismiss="cancelAction()"
     >
         <p
-            data-translate="ip_byoip_delete_block_description"
-            data-translate-values="{ link: serviceLink }"
+            data-translate="{{'ip_byoip_delete_block_description' + (!data.isDeleteByoipServiceAvailable ? '_info1' : '_info2')}}"
+            data-translate-values="{ link: (!data.isDeleteByoipServiceAvailable ? serviceLink : apiConsoleUrl) }"
```

Comment:
you can create small function in controller to return the good link and the good translation tag